src/main/java/youyihj/zenutils/api/liquid/MCLiquidTankProperties.java

```diff
@@ -0,0 +1,44 @@
+package youyihj.zenutils.api.liquid;
+
+import crafttweaker.annotations.ZenRegister;
+import crafttweaker.api.liquid.ILiquidStack;
+import crafttweaker.api.minecraft.CraftTweakerMC;
+import net.minecraftforge.fluids.capability.IFluidTankProperties;
+import stanhebben.zenscript.annotations.ZenClass;
+
+/**
+ * @author various.authors
+ */
+@ZenClass("mods.zenutils.LiquidTankProperties")
+@ZenRegister
+public class MCLiquidTankProperties implements ILiquidTankProperties {
```

Comment:
Remove the two annotations, since the interface it implements is exposed to zenscript. And move the class to `impl` package.

Reply:
Ah right, the interface exposes them. Cool!

Reply:
You forgot to move the class

Reply:
Ah OK! Question about that: why do `CrTLiquidHandler` and `CrTItemHandler` live in api? Should they also be in `impl`?

Reply:
Ah! Is it because they have zen annotations?